Unable to add to a project with ESLint 8

```npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: hv_2021@0.0.1
npm ERR! Found: eslint@8.5.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^8.3.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^7.10.0" from eslint-plugin-googlemaps@1.2.5
npm ERR! node_modules/eslint-plugin-googlemaps
npm ERR!   dev eslint-plugin-googlemaps@"*" from the root project```

--- a/package.json
+++ b/package.json
@@ -46,8 +46,7 @@
     "typescript": "^4.4.4"
   },
   "peerDependencies": {
-    "typescript": ">=3",
-    "eslint": "^7.10.0"
+    "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0"
   },
   "engines": {
     "node": "^12.13.0 || ^14.15.0 || >=16.0.0"

